_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Building Nested Enabling Teams
+ref: mandate
+lang: en
+status: posted
+categories: Work In Progress
+permalink: /building-enabling-directorate.html
+---
+
+## {{ page.title }}
+
+### Problem Statement
+
+In [Team Topologies](https://itrevolution.com/book/team-topologies/) they write that organizations should organize themselves by nesting the four team definitions.
+
+- Value Stream Aligned Teams
+- Platform Teams
+- Enabling Teams
+- Complex Subsystem Teams
+
+In our particular case, to use it as an example, the IT Strategy team is an enabling team. We are under a director who considers the directorate to be an enabling team.  Therefore the IT Strategy team is an enabling team within an enabling team, and has other sibling teams that are also enabling teams. That being the case, how would one synchronize activities of the entire directorate? Say the director was to offer their services to a team elsewhere in the organization. How best to manage and coordinate the work between the series of sub-enabling teams? That is the question this document is intending to answer.
+
+### Proposal
+
+#### Organizational
+
+##### Internal Interactions
+
+The teams may remain in their present functional structure for now, which aligns well with the governments expected structure for the sake of processes such as the Public Service Performance Management (PSPM). In our particular case, the teams will be as follows:
+
+- IT Strategy
```

Comment:
I think we should state that the IT Strategy team's enabling value is aligned to their mandate (e.g. de-risking the use of IT. For example, identifying waste and excessive work in progress within the overall IT value chain).

Reply:
"And establishing strategies to address those issues"

Reply:
I seem to recall a discussion where we suggested if folks are just re-wording or clarifying they should just modify the PR themselves. 

Discussion needed when folks would like to change the meaning of the idea/concept :)